packs/typeform/sensors/registration_sensor.py

```diff
@@ -0,0 +1,115 @@
+# Requirements:
+# See ../requirements.txt
+
+import httplib
+import requests
+import MySQLdb
+import MySQLdb.cursors
+
+from st2reactor.sensor.base import PollingSensor
+
+BASE_URL = 'https://api.typeform.com/v0/form'
+EMAIL_FIELD = "email_7723200"
+FIRST_NAME_FIELD = "textfield_7723291"
+LAST_NAME_FIELD = "textfield_7723236"
+SOURCE_FIELD = "textarea_7723206"
+NEWSLETTER_FIELD = "yesno_7723486"
+REFERER_FIELD = "referer"
+DATE_LAND_FIELD = "date_land"
+DATE_SUBMIT_FIELD = "date_submit"
+
+
+class TypeformRegistrationSensor(PollingSensor):
+    def __init__(self, sensor_service, config=None, poll_interval=180):
+        super(TypeformRegistrationSensor, self).__init__(
+            sensor_service=sensor_service,
+            config=config,
+            poll_interval=poll_interval)
+
+        self._trigger_pack = 'typeform'
+        self._trigger_ref = '.'.join([self._trigger_pack, 'registration'])
+
+        self.db = self._conn_db(host=self._config['db_host'],
+                                user=self._config['db_user'],
+                                passwd=self._config['db_pass'],
+                                db=self._config['db_name'])
+
+        self.url = self._get_url(self._config['form_id'],
+                                 self._config['api_key'],
+                                 self._config['completed'])
+
+    def setup(self):
+        pass
+
+    def poll(self):
+        registration = {}
+        api_registration_list = self._get_api_registrations()
+
+        for r in api_registration_list['responses']:
+            user = r['answers']
+            meta = r['metadata']
+            if self._check_db_registrations(user[EMAIL_FIELD]) is False:
+                registration['email'] = user[EMAIL_FIELD]
+                registration['first_name'] = user[FIRST_NAME_FIELD]
+                registration['last_name'] = user[LAST_NAME_FIELD]
```

Comment:
It's always best to access dict items like user.get(EMAIL_FIELD, default value if field doesn't exist). Otherwise these might throw KeyError.